library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -257,7 +257,3 @@ span.marker {
     margin: 0;
   }
 }
-html {
-    --sakai-font-family: roboto, arial, sans-serif;
-    --sakai-table-even-color: #f4f4f4;
-}
```

Comment:
I'd like to avoid multiple html/:root selectors that only exist for defining custom properties so I removed these and put the font-family in `sass/_defaults` and the table-even-color in `sass/themes/_light.scss`